crates/bevy_reflect/src/func/args/error.rs

```diff
@@ -0,0 +1,24 @@
+use alloc::borrow::Cow;
+
+use thiserror::Error;
+
+use crate::func::args::{ArgId, Ownership};
+
+/// An error that occurs when converting an argument.
```

Comment:
This would benefit from a doc link, probably to module docs: it's currently a dead-end and somewhat mysterious out of context.

Reply:
The module-level docs are pretty barebones/straight-to-the-point. So maybe I'll just link `argument` to `Arg`?

Reply:
Great, that works.